### PR TITLE
Test for "show all columns starting from column n" (~[n-])

### PR DESCRIPTION
**Pull request by pyllyukko**

### CODE REVIEW

File: t/feat_grep [hunk 1]
```diff
@@ -36,6 +36,18 @@ EXPECT='1020
 '
 run_test
 
+# columns starting from
+NAME='px~:1[1-]'
+FILE=malloc://1024
+ARGS=
+CMDS='
```
Comment (sivaramaaa) [inline t/feat_grep]: plss add a line BROKEN=1 if it the test case is broken

Comment (pyllyukko) [reply t/feat_grep]: <https://github.com/pyllyukko/radare2-regressions/commit/f3c0a8f0779fdf6de07882d715daf9e9bd845975>

**Comment by radare** [comment]
Conflicts

**Comment by pyllyukko** [comment]
I fscked up rebasin 😝 Anyways this test was already in #1124

**Pull request closed without merging**